.github/workflows/docker.yml

```diff
@@ -69,6 +69,19 @@ jobs:
           docker-images: true
           swap-storage: true
 
+      - name: Determine tag name
+        id: tag
+        shell: bash
+        run: |
+          BUILD_NUMBER="$(git rev-list --count HEAD)"
+          SHORT_HASH="$(git rev-parse --short=7 HEAD)"
```

Comment:
nitpick: these double quotes aren't necessary, command substitutions don't get expanded in variable assignments

Reply:
I basically copied this whole section from `.github/workflows/build.yml` happy to fix it if you want though

Reply:
Yup, would be nice to clean-up the extra quotes where not necessary